config/environments/development.rb

```diff
@@ -41,6 +41,20 @@
 
   config.action_mailer.perform_caching = false
 
+  # config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  config.action_mailer.delivery_method = :smtp
+  host = 'localhost:3000'
+  config.action_mailer.default_url_options = { :host => 'localhost:3000', protocol: 'http' }
+  config.action_mailer.smtp_settings = {
+    :address              => "smtp.gmail.com",
+    :port                 => 587,
+    :user_name            => "bogdan.szczerbak@gmail.com",
+    :password             => "pvjbzwlcafhrgypv",
```

Comment:
Keeping username and password like that is not safe and someone can use these credentials in order to send emails on your behalf.